README.md

```diff
@@ -21,67 +21,79 @@ The software is provided under a permissive "MIT" license (see below).
 
 * **Trading** - QSTrader will support live intraday trading using the Interactive Brokers API across a set of equities/ETFs.
 
-* **Performance Metrics** - QSTrader will support extensive strategy/portfolio performance measurement and equity curve visualisation via the Matplotlib and Seaborn visualisation libraries.
+# Installation and Example Usage
+
+QSTrader is in an extremely early alpha state at the moment and should only be used for exploratory backtesting research. The installation procedure is a little more involved than a standard Python package as it has not been added to the Python package repository yet.
 
-# Installation and Usage
+Ubuntu is the recommended platform on which to install QSTrader, but it will also work on Windows or Mac OSX under the Anaconda distribution (https://www.continuum.io/downloads).
 
-QSTrader is in an extremely early alpha state at the moment and should only be used for exploratory backtesting research.
+For those that wish to create their own virtual environment, the following steps are necessary to run a basic Buy And Hold strategy.
 
-1) Clone this git repository into a suitable location on your machine using the following command in your terminal: ```git clone https://github.com/mhallsmoore/qstrader.git```.
+An example virtual environment directory ```~/venv/qstraderp3``` has been used for this example. If you wish to change this directory then re-name it in the following steps.
 
-2) Create a virtual environment ("virtualenv") for the QSTrader code and utilise pip to install the requirements. For instance in a Unix-based system (Mac or Linux) you might create such a directory as follows by entering the following commands in the terminal:
+The following steps will create a virtual environment directory with Python 3 and then activate the environment:
 
 ```
-mkdir -p ~/venv/qstrader
-cd ~/venv/qstrader
-virtualenv .
+mkdir -p ~/venv/qstraderp3
+cd ~/venv/qstraderp3
+virtualenv --no-site-packages -p /usr/bin/python3 .
+source ~/venv/qstraderp3/bin/activate
 ```
 
-This will create a new virtual environment to install the packages into. Assuming you downloaded the QSTrader git repository into an example directory such as ```~/projects/qstrader/``` (change this directory below to wherever you installed QSTrader), then in order to install the packages you will need to run the following commands:
+At this point it is necessary to use pip to install QSTrader as a library and then manually install the requirements. The following steps will take some time as QSTrader relies on NumPy, SciPy, Pandas, Matplotlib as well as many other libraries and hence they will all need to compile:
 
 ```
-source ~/venv/qstrader/bin/activate
-pip install -r ~/projects/qstrader/requirements.txt
+pip install git+git://github.com/mhallsmoore/qstrader.git
```

Comment:
I think

    $ pip install git+git://

should be

    $ pip install git+https://